static/text-editor.less

```diff
@@ -2,6 +2,11 @@
 @import "octicon-utf-codes";
 @import "octicon-mixins";
 
+:root {
+  // Fixes specs
+  --editor-font-family: Menlo, Consolas, 'DejaVu Sans Mono', monospace;
```

Comment:
Looks like this change causes a [lint error](https://travis-ci.org/atom/atom/builds/342625366#L1137):

```
static/text-editor.css:1 Rule is empty. (empty-rules)
```

Could it be because the linter is old and doesn't know about custom properties?

Reply:
We might could add some dummy property. Just so the linter doesn't think it's empty, but this all feels like a big hack. :see_no_evil:

Reply:
Oh _wow_, `csslint` is the JSCS of CSS linters: It hasn't been updated in years.

I would _highly_ recommend looking into https://stylelint.io/ to replace that, it's based on PostCSS and is highly extensible. Think of it as the ESLint of style languages.